manifests/request.pp

```diff
@@ -211,7 +210,6 @@
 
   # Convert days to seconds for openssl...
   $renew_seconds = $renew_days*86400
-  notify { "acme renew set to ${renew_days} days (or ${renew_seconds} seconds) for cert ${name}": loglevel => debug }
```

Comment:
How does this `notify` leak secrets? It just contains renewal days/reconds and the cert name. I'd vote to keep it.

Reply:
As you can see in the screenshot, both notifies are reported through the puppetboard. Notifies should not be reported as changed, if no change was made. In general it's a bad idea to use this here, because it's harder to find anomalies if it always reports changes in monitoring tools like puppetboard for example. That's why we should fully remove it.

Reply:
Yeah, but only of you set loglevel to debug, which you should not use in normal operation. :blush:

Reply:
Somehow it's logged by the often used tool puppetboard. There is no option to change to debug. My suggestion is to remove it. Does this hurt your process somehow ?

Reply:
I don't know about Puppetboard, so I can't help you with that. But it's more likely that your Puppet _Agent_ is (mis)configured to use debug logging, hence the debug messages arrive in your PuppetDB / Puppetboard. The default `log_level` is `notice` and like I said `debug` should not be used:
https://www.puppet.com/docs/puppet/8/configuration#log-level

The message I want to keep is useful for debugging and doesn't leak any secrets.

Reply:
As in documentation says https://www.puppet.com/docs/puppet/8/types/notify.html - "It's important to note that the notify resource type is not idempotent." You should not do it that way. Its stored in PuppetDB, that why it's printed out. I would suggest to create a fact for this, if its really required. Maybe would be grate to add also expire date, and other information's. What do you think?

Reply:
Don't enable debug logging if you don't want to see debug messages. :shrug: 
Let's agree to disagree. I've cherry-picked https://github.com/markt-de/puppet-acme/commit/6e40b8eaab98396647efc52b66575e02c30611d3 to fix the secret leakage.